randovania/games/dread/logic_database/Ghavoran.txt

```diff
@@ -2061,46 +2061,69 @@ Extra - asset_id: collision_camera_026
+                          Flash Shift or Use Spin Boost
           Any of the following:
               # Requirements for killing Golzuna
-              Shoot Charge Beam
+              All of the following:
+                  # Charge Beam methods
+                  Shoot Charge Beam
+                  Any of the following:
+                      # Plasma, Wave, or Power
+                      Combat (Intermediate) or Beam Damage 120 or Beam Damage 240
+                      # Wide
+                      Combat (Beginner) and Beam Damage 75
+              # Shinesparks only
               Speed Booster and Combat (Advanced)
               All of the following:
-                  Combat (Intermediate)
+                  # Missile methods
                   Any of the following:
                       All of the following:
                           # Minimum: 35 Ice Missiles
                           Shoot Super Ice Missile
                           Any of the following:
                               Missiles ≥ 40
                               # Minimum: 18 Ice Missiles + 1 Shinespark
-                              Missiles ≥ 20 and Speed Booster
+                              Missiles ≥ 20 and Speed Booster and Combat (Intermediate)
                       All of the following:
                           # Minimum: 94 Super Missiles
-                          Shoot Super Missile
+                          Combat (Beginner) and Shoot Super Missile
                           Any of the following:
                               Missiles ≥ 100
                               # Minimum: 47 Super Missiles + 1 Shinespark
-                              Missiles ≥ 50 and Speed Booster
+                              Missiles ≥ 50 and Speed Booster and Combat (Intermediate)
                       All of the following:
                           # Minimum: 280 Normal Missiles
+                          Combat (Intermediate)
                           Any of the following:
                               Missiles ≥ 300
                               # Minimum: 140 Normal Missiles + 1 Shinespark
                               Missiles ≥ 150 and Speed Booster
+                      All of the following:
+                          # Minimum: 96 Storm missiles
+                          Storm Missile
+                          Any of the following:
+                              Missiles ≥ 108
+                              # Mimimum: 48 Storm missiles + 1 shinespark
+                              Missiles ≥ 60 and Speed Booster and Combat (Intermediate)
+              Any of the following:
+                  # Bomb methods
+                  # Normal Bombs
+                  Combat (Intermediate) and Lay Bomb
+                  # Cross Bombs
+                  Combat (Beginner) and Lay Cross Bomb
           Any of the following:
               # Energy Requirements
-              Combat (Intermediate) or Normal Damage ≥ 699
+              Normal Damage ≥ 699
```

Comment:
There's probably room for having a combat level that doesn't require any health, though I think it could reasonably he as high as Expert.

Reply:
seeing as you can't survive being tapped by Golzuna with 0 tanks even with gravity, I don't see it as doable.

Reply:
My thinking is that there are no paths that require you to take damage, so the damage requirements are here only to act as leeway so that players don't need to perfectly avoid damage.